packages/markdown-it-cjk-friendly/src/index.ts

```diff
@@ -60,7 +60,7 @@ function nonEmojiGeneralUseVS(uc: number) {
   return uc >= 0xfe00 && uc <= 0xfe0e;
 }
 
-export default function markdownItCjFriendlyPlugin(md: MarkdownIt): void {
+function markdownItCjFriendlyPlugin(md: MarkdownIt): void {
```

Comment:
How about this to allow both default and named imports?

```suggestion
export function markdownItCjFriendlyPlugin(md: MarkdownIt): void {
```